src/analysis/calculus/fderiv.lean

```diff
@@ -2462,3 +2461,158 @@ lemma differentiable.restrict_scalars (h : differentiable 𝕜' f) :
 λx, (h x).restrict_scalars 𝕜
 
 end restrict_scalars
+
+/-!
+### Multiplying by a complex function respects real differentiability
+
+Consider two functions `c : E → ℂ` and `f : E → F` where `F` is a complex vector space. If both
+`c` and `f` are differentiable over `ℝ`, then so is there product. This paragraph proves this
```

Comment:
```suggestion
`c` and `f` are differentiable over `ℝ`, then so is their product. This paragraph proves this
```

Reply:
Thanks, and sorry! (To me, this was the kind of mistake that only native speakers could do, because when you learn a foreign language you start with the grammar, so "their" and "there" are completely different. I don't know what that says of my English level :-)